graph/impl/src/main/java/org/jboss/windup/graph/GraphContextImpl.java

```diff
@@ -272,10 +274,19 @@ private TitanGraph initializeTitanGraph()
         // the performance decrease seems to be minimal.
         conf.setProperty("storage.berkeleydb.cache-percentage", 1);
 
+        conf.setProperty("storage.berkeleydb.lock-mode", LockMode.READ_UNCOMMITTED);
+        conf.setProperty("storage.berkeleydb.isolation-level", BerkeleyJEStoreManager.IsolationLevel.READ_UNCOMMITTED);
+
         // Increase storage write buffer since we basically do a large bulk load during the first phases.
         // See http://s3.thinkaurelius.com/docs/titan/current/bulk-loading.html
         conf.setProperty("storage.buffer-size", "4096");
 
+        conf.setProperty("storage.transactions", false);
```

Comment:
Yes, it disables all transactions. I don't think we rely on rollback behavior in order to avoid corruption, so I don't think this causes issues for us. I could be wrong, though, this obviously needs some testing.